hypothesis-python/tests/cover/test_health_checks.py

```diff
@@ -109,21 +110,21 @@ def do_draw(self, data):
 
 def test_filtering_most_things_fails_a_health_check():
     @given(fails_regularly())
-    @settings(database=None, phases=no_shrink)
+    @settings(database=None, phases=no_shrink, suppress_health_check=())
     def test(x):
-        pass
+        if current_build_context().data.provider.avoid_realization:
+            pytest.skip("symbolic backends can filter efficiently!")
 
```

Comment:
Should we instead hardcode a check against the crosshair backend, or only run on the hypothesis backend? `avoid_realization` is correlated with but not quite the same as a backend filtering efficiently.

Reply:
I'm basically assuming that "`avoid_realization` means symbolic execution", so while it's not the same thing I think it's likely to be the right call.